binder/views.py

```diff
@@ -35,6 +35,25 @@
 from .json import JsonResponse, jsonloads
 
 
+def get_joins_from_queryset(queryset):
+	compiler = queryset.query.get_compiler(queryset.db)
+	connection = connections[queryset.db]
+	lines = set()
+	for alias in queryset.query.alias_map.values():
+		line, params = alias.as_sql(compiler, connection)
+		assert not params
+		lines.add(line)
+	return lines
+
+
+def q_get_flat_filters(q):
```

Comment:
Maybe add comments here on how it works/ and maybe a small docstring what it returns?